### PR TITLE
HAL-786 Make HAL testsuite modular

### PR DESCRIPTION
**Pull request by zebrik**
Motivation: ability to clearly separate configuration for e.g. RBAC related tests

### CODE REVIEW

**Comment by zebrik** [comment]
@rhatlapa please review
@mkrajcov @honza-kasik  @cypo721 @hpehl @heiko-braun FYI

File: basic/pom.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <version>0.0.2-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>hal-testsuite-basic</artifactId>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.jboss.as</groupId>
+      <artifactId>hal-testsuite-common</artifactId>
+      <version>${project.version}</version>
```
Comment (rhatlapa) [inline basic/pom.xml]: Putting the version to dependency management in parent might be better.

File: rbac/pom.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    <version>0.0.2-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>hal-testsuite-rbac</artifactId>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.jboss.as</groupId>
+      <artifactId>hal-testsuite-common</artifactId>
+      <version>${project.version}</version>
```
Comment (rhatlapa) [inline rbac/pom.xml]: The same as above

**Comment by zebrik** [comment]
I agree that putting the version to dependency management in parent would be better.
Force pushed to pom.xml basic/pom.xml rbac/pom.xml

**Comment by zebrik** [comment]
Just have updated README yet - info about modules.

**Comment by rhatlapa** [comment]
Looks good to me, merging.

**Pull request merged**